pkgs/build-support/fetchsrht/default.nix

```diff
@@ -0,0 +1,29 @@
+{ fetchzip, lib }:
+
+{ user ? null
+, group ? null
```

Comment:
`group` is unused.

Reply:
I generally think the `user`, `group` thing is an unnecessary complications and handling conditional args is always a nightmare to maintain because the expressions get obfuscated. With the `group` prefix potentially not being stable this sounds even worse.

Also the quality of life improvement of `user = "foo"` over `owner = "~foo"` isn't really worth it imo.

Reply:
I guess you are right, I overthought that a lot…